src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts

```diff
@@ -545,7 +553,11 @@ export const ApplicantDetails = (
     if (sessionApplicantData[applicant]['liveInRefuge'] === YesOrNo.YES) {
       newApplicantData.push({
         key: keys['c8RefugeDocument'],
-        value: sessionApplicantData[applicant]['refugeConfidentialityC8Form']?.['document_filename'],
+        keyName: `${sessionApplicantData[applicant]['applicantFirstName']}${sessionApplicantData[applicant]['applicantLastName']}c8RefugeDocument`,
```

Comment:
Can have the static names

Reply:
Changed to c8RefugeDocument-applicant-{applicantIndex} or c8RefugeDocument-otherPerson-{otherPersonIndex}